ocaml/testsuite/tests/typing-layouts-float64/basics.ml

```diff
@@ -22,3 +22,18 @@ Line 1, characters 15-26:
 Error: Layout float64 is used here, but the appropriate layouts extension is not enabled
 |}]
 
+(* CR layouts: The below test checks that we give an acceptable error for cases
+   where a float64 was used from a library in a file where the user forgot to
+   enable the relevant extension.  It can be deleted when float64 is on by
+   default (though at that time we should add tests for explicitly disabling the
+   layouts extension). *)
+let f x = Stdlib__Float_u.sin x
+[%%expect{|
+Line 1, characters 6-31:
+1 | let f x = Stdlib__Float_u.sin x
+          ^^^^^^^^^^^^^^^^^^^^^^^^^
+Error: Non-value layout float64 detected as sort for type float#,
+       but this requires extension layouts_beta, which is not enabled.
+       If you intended to use this layout, please add this flag to your build file.
```

Comment:
In the past I've been advised against leaking details of our build system into the compiler.  I don't personally have a strong preference, but since there's disagreement and it's less work to just merge, I guess I'll do that.

(There will be many chances to revise this message as we add more layouts and move them between the levels)